tests/acceptance/variant_browser_bottom_panel_tests.js

```diff
@@ -133,10 +133,10 @@ module.exports = {
                         });
                         //check study title links
                         rows[i].findElement(By.className("study_link")).getAttribute('href').then(function(text){
-                            assert(text.split("?")[1]).matches(/^eva-study\=PRJ[A-Z0-9]+$/);
+                            assert(text.split("?")[1]).matches(/^eva-study\=[A-Z0-9-_]+$/);
                         },function(err) {});
                         rows[i].findElement(By.className("project_link")).getAttribute('href').then(function(text){
-                            assert(text.split("?")[1]).matches(/^eva-study\=PRJ[A-Z0-9]+$/);
+                            assert(text.split("?")[1]).matches(/^eva-study\=[A-Z0-9-_]+$/);
```

Comment:
An example of a study ID (BATCH-PTLR8-DEVELOPMENT) with hyphens in them. I think there are some studies (which I don't have an immediate example for) which have underscores in them too.
![image](https://user-images.githubusercontent.com/25152678/66118871-5515e500-e5cf-11e9-852c-c70a10893904.png)

Reply:
OK, now this worries me for another reason: all studies which I'm trying to import from EVAPRO actually have their accessions starting from either ERP or SRP. If we have studies with other name patterns, _where_ are they? Won't we forget to import them when migrating to the new metadata service? And since they have never been included into a test run, what undetected problems might they harbour?

Reply:
small note: for EVA, "study" and "project" are the same, and both are ENA's "project". EVA does not use ENA's studies. We also use ENA's "analysis" as EVA's "files" (approximately).

Reply:
Created an AMP ticket to address this: https://www.ebi.ac.uk/panda/jira/browse/T2D-361